docs/reference/go-style-annotation-property-practice.md

```diff
@@ -0,0 +1,18 @@
+| Synonyms                 | http://www.geneontology.org/formats/oboInOwl#has_exact_synonym, http://www.geneontology.org/formats/oboInOwl#has_broad_synonym, http://www.geneontology.org/formats/oboInOwl#has_narrow_synonym, http://www.geneontology.org/formats/oboInOwl#has_related_synonym                              | N                             | No limit     | Synonyms of the term.                                                                                              | Free text                           | database_cross_reference: reference material in which the synonymn is used                    | See [synonyms documentation](../reference/synonyms-obo.md) for guide on using synonyms                                                                     |
+| Comments                 | rdfs:comment                                                                                                     | N                             | Max 1        | Comments about the term, extended descriptions that might be useful, notes on modelling choices, other misc notes. | Free text                           | database_cross_reference: reference material relating to the comment                          | See [documentation on comments](../explanation/term-comments.md) for more information about comments                                                             |
+| Editor note              | IAO:0000116                                                                                                      | N                             | Max 1        | A note that is not relevant to front users, but might be to editors                                                | Free text                           | database_cross_reference: reference material relating to the note                             |                                                                                                                                |
+| Subset                   | [http://www.geneontology.org/formats/oboInOwl#inSubset](http://www.geneontology.org/formats/oboInOwl#inSubset)   | N                             | No limit     | A tag that marks a term as being part of a subset                                                                  | subset_property                    | None                                                                                            | See [Slim documentation](../howto/add-new-slim.md) for more information on subsets    |
```

Comment:
Example value? I see people doing that wrong all the time. The subset must be an annotation property that is a subproperty of subset_property

Reply:
I think an example wouldn't be useful, instead I'll add a comment there to make sure people look into the guide to select it properly.